### DISCUSSION TITLE
News Release文件注释了模块名称

### DISCUSSION THREAD

**Discussion by kstonfr** [NSRingo/.github#3] [Q&A 问答] [answered]
3.1.5: https://github.com/NSRingo/News/releases/download/v3.1.5/News.snippet

![CleanShot 2024-10-18 at 18 48 51@2x](https://github.com/user-attachments/assets/4fec1fab-e5b7-402d-979c-d943d97ebe93)

**Comment by VirgilClyne** [comment, marked as answer]
`Quantumult X`的`snippet`不可携带段落标签
请参阅学习官方示例：
https://github.com/crossutility/Quantumult-X/blob/master/sample-import-rewrite.snippet
